assets/css/index.css

```diff
@@ -195,7 +197,7 @@ rh-global-footer {
         }
         &.active {
           & a {
-            background: var(--rh-color-surface-lighter);
+            background: gray;
```

Comment:
These should be reverted, we should use the token values for brand consistency

Reply:
fixed